lib/view.js

```diff
@@ -24,6 +24,7 @@ export default class AdvancedOpenFileView {
         this.emitter = new Emitter();
         this.cursorIndex = null;
         this._updatingPath = false;
+        this.prevPath = "";
```

Comment:
eslint is throwing warnings due to double quotes, but apparently the TravisCI build doesn't actually fail when eslint does. Huh.

Could we use single quotes everywhere in this PR?

Reply:
Oh, sure. I'm used to the opposite, so I just missed that.